reference/v8.json

```diff
@@ -1861,6 +1904,9 @@
       "sdk-support": {
         "basic functionality": {
           "js": "0.27.0"
+        },
+        "data-driven styling": {
+          "js": "Not supported"
```

Comment:
Ah, whoops. Yeah, `fill-extrusion-opacity` is probably never going to support data-driven styling, for the reasons outlined in #545. I'll see if I can tweak the template to make this work.